components/Footer.tsx

```diff
@@ -52,19 +77,19 @@ export default function Footer() {
           <div className="grid grid-cols-2 md:flex md:gap-16 font-medium">
             <nav className="flex flex-col space-y-3">
               {navigation.map((item) => (
-                <Link
+                <button
                   key={item.name}
-                  href={item.href}
-                  className="text-primary-900 hover:text-primary-600 transition-colors text-sm"
+                  onClick={() => handleNavigation(item.href)}
+                  className="text-primary-900 hover:text-primary-600 transition-colors text-sm text-left"
                 >
                   {item.name}
-                </Link>
+                </button>
```

Comment:
It's alright if you remove the smooth scroll feature, but don't change this from link to button. Reason being -> 
1. you won't be able to see a link preview when you hover over it
2. you wont be able to open the link in a new tab. 

This will hinder UX